pkg/cli/util/breakingchange/upgradehandlerto0.7.go

```diff
@@ -507,3 +531,83 @@ func (u *upgradeHandlerTo7) transformConfigMap(dynamic dynamic.Interface, obj un
 	}
 	return nil
 }
+
+func (u *upgradeHandlerTo7) transformQdrantCd(dynamic dynamic.Interface, obj unstructured.Unstructured) error {
```

Comment:
The CD will be upgrade to new version, so it maybe not need to transform. And the qdrant-1.5.0 cv will be created, and the older cv qdarnt-1.1.0 will be deleted. So I suggest to modify the qdrant cluster to use the new cv qdrant-1.5.0.

Reply:
Get It！